release/jobs/rds-broker/spec

```diff
@@ -37,6 +37,8 @@ properties:
     default: "cf"
   rds-broker.master_password_seed:
     description: "Secret seed to be used when generating the master RDS DB password"
+  rds-broker.aws_tag_cache_seconds
+    description: "Cache expiry time (in seconds) of AWS Tags cache"
```

Comment:
I think you might need to set a default value here, or it'll be the empty string, which will mess up the JSON config file

Reply:
Ugh I was hoping it'd be `null` and fall through as zero.